rackcheck/info.rkt

```diff
@@ -3,7 +3,8 @@
 (define version "2.0.1")
 (define collection "rackcheck")
 (define deps '("base"
-               "rackunit-lib"))
+               "rackunit-lib"
```

Comment:
This dependency is probably no longer needed at runtime (you can run `raco setup --tidy --check-pkg-deps --unused-pkg-deps --pkgs rackcheck` to find out for sure).

Reply:
Indeed it was not needed `raco setup --tidy --check-pkg-deps --unused-pkg-deps --pkgs rackcheck` printed:

```
raco setup: unused dependencies detected
  for package: "rackcheck"
  on packages:
   "rackcheck-lib" for run
   "rackunit-lib" for run
   "testing-util-lib" for run
 ```
 
After change (https://github.com/Bogdanp/rackcheck/pull/10/commits/c2b689f5fbae0777ae12617f07d0612370a72a17):

```
raco setup: unused dependency detected
  for package: "rackcheck"
  on package:
   "rackcheck-lib" for run
```
I assume this is OK as we want dependency between `rackcheck` and `rackcheck-lib`.

Reply:
A small follow-up on this: I had forgotten that there is an `implies` field as well. Using `implies` instead of `update-implies` gets rid of the unused dependency warning (by implying a stronger link between the packages (documented [here](https://docs.racket-lang.org/pkg/metadata.html))).